### ISSUE TITLE
v2 branch

### ISSUE THREAD

**Issue by mspykerez** [andrebrait/1g1r-romset-generator#27]
I notice that there's a v2 branch available to download with the last commit marked 2 weeks ago. Can it be confirmed that this branch is good for testing? It seems better than waiting for a full release... its been over a month now! I will use it if confirmed good enough for use and the full release is more than 1 week away else I will wait.

**Comment by andrebrait** [comment]
No, v2 is not ready at all. I was busy with a course the oast few weeks. I should be able to start working on it again now. A release is most definitely more than a week away, though.

**Comment by mspykerez** [comment]
OK thx anyway I will wait!

**Comment by andrebrait** [comment]
Just giving you an update, I'll resume working on v2 after fixing #23 in v1.9 :)

**Comment by mspykerez** [comment]
Any news ? it's been half a year and still no v2 release...

**Comment by andrebrait** [comment]
So, v2 is not going to happen anymore.

I've started a new tool which I plan to release this quarter. https://github.com/andrebrait/DATROMTool

That is a lot more flexible than this one. It already works fine, but I haven't written instructions yet.
